ui/src/app/base/components/node/networking/FabricColumn/FabricColumn.tsx

```diff
@@ -5,38 +5,33 @@ import DoubleRow from "app/base/components/DoubleRow";
 import LegacyLink from "app/base/components/LegacyLink";
 import baseURLs from "app/base/urls";
 import fabricSelectors from "app/store/fabric/selectors";
-import machineSelectors from "app/store/machine/selectors";
-import type { Machine } from "app/store/machine/types";
 import type { RootState } from "app/store/root/types";
-import type { NetworkInterface, NetworkLink } from "app/store/types/node";
+import type { NetworkInterface, NetworkLink, Node } from "app/store/types/node";
 import { getInterfaceFabric, isBondOrBridgeParent } from "app/store/utils";
 import vlanSelectors from "app/store/vlan/selectors";
 import { getVLANDisplay } from "app/store/vlan/utils";
 
 type Props = {
   link?: NetworkLink | null;
   nic?: NetworkInterface | null;
-  systemId: Machine["system_id"];
+  node: Node;
 };
 
-const FabricColumn = ({ link, nic, systemId }: Props): JSX.Element | null => {
-  const machine = useSelector((state: RootState) =>
-    machineSelectors.getById(state, systemId)
-  );
+const FabricColumn = ({ link, nic, node }: Props): JSX.Element | null => {
   const fabricsLoaded = useSelector(fabricSelectors.loaded);
   const fabrics = useSelector(fabricSelectors.all);
   const vlan = useSelector((state: RootState) =>
     vlanSelectors.getById(state, nic?.vlan_id)
   );
   const vlans = useSelector(vlanSelectors.all);
-  if (!machine) {
+  if (!node) {
     return null;
   }
```

Comment:
This might be unnecessary if `node` is a required prop, otherwise `Props` should be updated to allow falsy `node` values.

Reply:
Oh yeah, good call. Thanks.